CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Blocks of changes will separated by version increments.
 
 ## **[Unreleased]**
 
+- [#858](https://github.com/wasmerio/wasmer/pull/858) Fix panic when `loader` option associated with wasm module without exported `_start` function
```

Comment:
It would be good to clarify when the panics are specific to using wasmer as a binary vs wasmer as a library.  Panics converted into errors in the binary are minor aesthetic issues in practice.  However panics from use of the library may be indicative of an underlying issue and deserve much more increased scrutiny in my opinion.  I think it's good to fix them in either case because it probably makes automatic detection of panics easier, but if I wanted to skim through the changelog to review the caught panics in more depth, I'd feel that looking at the ones in `src/bin/wasmer.rs` are mostly noise.

Please let me know if that doesn't make sense to you or if you disagree.

Thanks!

Reply:
Sure, totally agree with you ;)
i will make it more clear

Reply:
please check new CHANGELOG.md